apps/sample-app/src/components/Profile.tsx

```diff
@@ -0,0 +1,65 @@
+import { Text } from '@fluentui/react-components';
+import { AuthProvider, AuthProviderCallback, Client, Options } from '@microsoft/microsoft-graph-client';
+import { Calendar, User } from '@microsoft/microsoft-graph-types';
+import React from 'react';
+
+import { MainPage } from './MainPage';
+import { MeetingList } from './Meetings';
+import { PeopleAvatarList } from './PeopleAvatars';
+import { getDates } from './utils';
+
+interface ProfileContentProps {
+  accessToken: string;
+}
+
+export const ProfileContent: React.FC<ProfileContentProps> = (props: ProfileContentProps) => {
+  const { accessToken } = props;
+  const [userInfo, setUserInfo] = React.useState<User>();
+  const [calendar, setCalendar] = React.useState<Calendar>();
+
+  React.useEffect(() => {
+    (async () => {
+      const authProvider: AuthProvider = (callback: AuthProviderCallback) => {
+        callback(undefined, accessToken);
+      };
+      console.log('starting info');
```

Comment:
probably good to remove this console.log unless it's giving consumers of these samples some info they need to understand what's going on